src/test/java/org/testeditor/fixture/web/drivermanager/InternetExplorerTest.java

```diff
@@ -0,0 +1,138 @@
+import org.openqa.selenium.ie.InternetExplorerDriver;
+import org.openqa.selenium.remote.DesiredCapabilities;
+
+public class InternetExplorerTest {
+
+    private WebDriver driver;
+
+    private static String proxyHost;
+    private static String proxyUser;
+    private static String proxyPassword;
+    private static String testSiteUrl;
+    private static String searchfieldID;
+    private static String searchString;
+    private static String firstSiteTitle;
+    private static String xPathResult;
+    private static String resultSiteTitle;
+
+    @BeforeClass
+    public static void setupClass() throws IOException {
+        assumeWindowsPresent();
+        BrowserManager browserManager = InternetExplorerDriverManager.getInstance();
+        browserManager.proxy(proxyHost);
+        browserManager.proxyUser(proxyUser);
+        browserManager.proxyPass(proxyPassword);
+        browserManager.setup();
+    }
+
+    @Before
+    public void setupTest() throws IOException {
+        DesiredCapabilities capabilities = DesiredCapabilities.internetExplorer();
+        capabilities.setCapability(InternetExplorerDriver.INTRODUCE_FLAKINESS_BY_IGNORING_SECURITY_DOMAINS, true);
+        capabilities.setCapability("requireWindowFocus", true);
+
+        driver = new InternetExplorerDriver(capabilities);
+    }
+
+    @After
+    public void teardown() {
+        if (driver != null) {
+            driver.quit();
+        }
+    }
+
+    @Test
+    public void test() throws InterruptedException {
+        driver.get(testSiteUrl);
+        String title = driver.getTitle();
+        Assert.assertEquals(firstSiteTitle, title);
+        WebElement webElement = driver.findElement(By.id(searchfieldID));
+        webElement.sendKeys(searchString);
+        webElement.sendKeys(Keys.ENTER);
+        waitSeconds(1);
+        webElement = driver.findElement(By.xpath(xPathResult));
+        webElement.click();
+        waitSeconds(1);
+        title = driver.getTitle();
+        Assert.assertEquals(resultSiteTitle, title);
+
+    }
+
+    public void waitSeconds(long timeToWait) throws InterruptedException {
+        Thread.sleep(timeToWait * 1000);
+    }
+
+    /**
+     * To execute these Tests it is necessary to provide a file named
+     * "ie_test.properties" specified in the variable "testPropertyPath" below.
+     * The content of the property file should contain following key value
+     * pairs.
+     * 
+     * <pre>
+     * PROXY_HOST=your.proxy.system.here
+     * PROXY_USER=yourProxyUsernameHere
+     * PROXY_PASSWORD=yourProxyPasswordHere
+     * TEST_SITE_URL=yourTestSiteUrlHere
+     * SEARCH_FIELD_ID=yourSearchFieldToTypeInHere
+     * SEARCH_STRING=yourSearchStringHere
+     * FIRST_SITE_TITLE=yourFirstWebsiteTitle
+     * XPATH_RESULT=yourXpathEntryResult
+     * RESULT_SITE_TITLE=yourWebsiteTitleAfterSearch
+     * </pre>
+     * 
+     * @throws IOException
+     */
+    public static void assumeWindowsPresent() throws IOException {
```

Comment:
this function should be named 'assumeWindowsPresentAndSetupProxyAndReadPropertiesFile' which obviously is not wanted, right?